### ISSUE TITLE
Hovering over the Animate Guardian Skill crashes PoB

### ISSUE THREAD

**Issue by ToastdasToast** [Openarl/PathOfBuilding#1673]

**Comment by ToastdasToast** [comment]
https://imgur.com/DxFfTUh
Build: https://pastebin.com/kQNz7LtC
PoB version Latest: 1.4.151

**Comment by hateborne** [comment]
Ok, so it's localized to your equipment. I'm not sure why, but still digging. If you hit 'Delete All' on the Items tab, the skill works fine. I suspected it may be a simple limitation on number of effects/auras, but I was incorrect. The only way this works normally is deleting all of the Items. 

Have you created a custom item or modified an item through the custom crafting/editing?

**Comment by ToastdasToast** [comment]
Yes I probably did edit a thing or two, the character was an import from last league around mid league, I probably edited the bow once or twice since then. I haven't been able to replicate the issue since then on a different character. Upon cancelling the "attempt to index field 'weaponData' (a nil value)" error, I get another OnFrame Error 
![image](https://user-images.githubusercontent.com/28730658/64636622-5efa5d00-d402-11e9-9ed9-a0f1a869530e.png)
Which cant be closed unless i restart PoB (F5 or just Close->ReOpen)

**Comment by Openarl** [comment]
Well.. your default item set has pretty much every item equipped in the wrong slot (Abyss Jewel as a main hand, boots as off hand, etc). Naturally the program isn't impressed by it! I have no idea how your build managed to get into that state, but deleting that item set will fix the crash.

**Comment by ToastdasToast** [comment]
Displays it for me just fine, but in case it is that way, no wonder PoB
went like "nah dude not happening".
Now I'm curious how that happened
Thanks for the reply

Openarl <notifications@github.com> schrieb am So., 15. Sep. 2019, 5:29
vorm.:
